SwiftCompilerSources/Sources/Optimizer/PassManager/PassRegistration.swift

```diff
@@ -87,6 +87,8 @@ private func registerSwiftPasses() {
   registerPass(deadStoreElimination, { deadStoreElimination.run($0) })
   registerPass(redundantLoadElimination, { redundantLoadElimination.run($0) })
   registerPass(earlyRedundantLoadElimination, { earlyRedundantLoadElimination.run($0) })
+  registerPass(diagnoseLifetimeDependencePass, {
+    diagnoseLifetimeDependencePass.run($0) })
```

Comment:
Fixed.

But running swift-format turns these two lines into five lines, so I don't plan on using it:
```
+  registerPass(
+    diagnoseLifetimeDependencePass,
+    {
+      diagnoseLifetimeDependencePass.run($0)
+    })
```

This is something I'll just need to fix manually case-by-case. Or whoever is bothered by it can fix it.

My editor panes are always 80 columns for important reasons, so code I'm very actively working on will start out that way. Every major project I've worked on has eventually settled on 80 cols after a long debate once everyone finally understood all the issues. Namely that code needs to be vertically oriented to be broadly readable and maintainable, scaling to a large toolset. This includes LLVM, Swift compiler, and all the standard Swift packages.